src/host_interface/virtio_netdev.c

```diff
@@ -227,14 +227,15 @@ static void virtio_net_fd_net_poll_hup(uint8_t netdev_id)
     close(nd_fd->pipe[1]);
 }
 
+// FIXME not called from anywhere
 /*
  * Function to close the net device
  */
-static void virtio_net_fd_net_free(uint8_t netdev_id)
-{
-    struct netdev_fd* nd_fd = get_netdev_fd_instance(netdev_id);
-    close(nd_fd->fd);
-}
+//static void virtio_net_fd_net_free(uint8_t netdev_id)
+//{
+//    struct netdev_fd* nd_fd = get_netdev_fd_instance(netdev_id);
+//    close(nd_fd->fd);
+//}
```

Comment:
Perhaps it's better to leave this in and keep the warning? This needs to be fixed.

Reply:
I don't want to generally allow unused code, so either we comment it out, remove it, or we temporarily disable the warning using gcc pragmas. Or we actually call the function...

Reply:
commenting out seems like a bad idea. its noise.

delete or call it would be my "vote".

Reply:
I'll leave it commented out for now. I tried to understand if it's safe to call it from `net_dev_remove` but this is called from the enclave and I'm not sure if host threads would still need it. If anyone wants to fix it, please send a follow-up PR :)